modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -1003,6 +1005,132 @@ public void testBuildTemplateServiceBuilderNestedPath() throws Exception {
 			":modules:nested:path:sample");
 	}
 
+	@Test
+	public void testBuildTemplateServiceBuilderVersions() throws Exception {
+		String projectPath = ":modules:nested:path:sample";
+
+		String name = "sample";
+
+		String packageName = "com.test.sample";
+
+		File workspaceProjectDir = _buildTemplateWithGradle(
```

Comment:
There is no need to test workspace version vs a maven.  Just compare standalone Gradle version to standalone maven version.